docs/user/README.md

```diff
@@ -16,9 +16,9 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
 1. Install [cURL](https://curl.se/) as the *root* user: `pkg install curl`
 
 ### Windows
-1. Install a supported version of [Visual Studio](https://visualstudio.microsoft.com/vs/older-downloads/) with Visual C++:
-   - [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products) ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
-   - [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
+1. Install the Microsoft Visual C++ Prerequisites for Rust and Python 3:
+   - Microsoft Visual C++ Prerequisites for Rust: see [The rustup book](https://rust-lang.github.io/rustup/installation/windows-msvc.html).
```

Comment:
The linked guide mentions that Rust supports "Visual Studio 2013 and later" but since we have had issues using any other version than 2022, should we be explicit and say that they should install Visual Studio 2022? The walk-through in the guide uses 2022 so it should be fine for new installs but a user might think that an already installed version is sufficient.

Reply:
Generally I would prefer not to mention specific version number as we then have to continuously update them and since VS 2022 is used in the guide I decided not to mention a specific version in our documentation. What do you think @chrthomsen?

Reply:
Maybe we could simply add something like "the latest version" to Line 31 so it becomes "1. Install the latest version of the Microsoft Visual C++ Prerequisites for Rust and Python 3 "?

Reply:
Even though it was found to work with VS2019 I still think it makes sense to add "the latest versions" since it should still be preferable.

Reply:
I agree with adding "the latest versions"